ui/app/templates/components/mount-backend-form.hbs

```diff
@@ -1,96 +1,124 @@
 <PageHeader as |p|>
   <p.levelLeft>
-    <h1 class="title is-3" data-test-mount-form-header=true>
+    <h1 class="title is-3" data-test-mount-form-header="true">
       {{#if showConfig}}
-        {{#with (find-by 'type' mountModel.type mountTypes) as |typeInfo|}}
-          <ICon @size=24 @glyph={{concat "enable/" (or typeInfo.glyph typeInfo.type)}} @class="has-text-grey-light" />
+        {{#with (find-by "type" mountModel.type mountTypes) as |typeInfo|}}
```

Comment:
Does the above line need to be `showEnable`? (I thought I remembered the bool changing in the JS)

Reply:
just fixed that. you're right!